x/ratelimit/types/msgs.go

```diff
@@ -0,0 +1,195 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// Msg type for MsgAddQuota
+const TypeMsgAddQuota = "add_quota"
+
+var _ sdk.Msg = &MsgAddQuota{}
+
+func NewMsgAddQuota(name string, maxPercentSend uint64, maxPercentRecv uint64, durationMinutes uint64) *MsgAddQuota {
+	return &MsgAddQuota{
+		Name:            name,
```

Comment:
Should the `Creator` field be included in these `NewMsg` functions?

Reply:
Yep, added it!